app/next-client-app/api/scanreports.ts

```diff
@@ -48,7 +48,9 @@ export async function getScanReports(
  * @param id The Id of the Scan Report
  * @returns A object with a list of the users permissions.
  */
-export async function getScanReportPermissions(id: number): Promise<{}> {
+export async function getScanReportPermissions(
+  id: string
+): Promise<{ permissions: string[] }> {
```

Comment:
Don't we have permissions type?

Reply:
Yes, I was using the wrong one. Already fixed this. Thanks for posting it!